test/suite/test_backup29.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import os, re
+from wtscenario import make_scenarios
+from wtbackup import backup_base
+
+# test_backup29.py
+#    Test interaction between restart, checkpoint and incremental backup.
+#
+class test_backup29(backup_base):
+    #conn_config = 'create,verbose=(backup)'
+    create_config = 'allocation_size=512,key_format=i,value_format=S'
+    # Backup directory name
+    dir='backup.dir'
+    incr_dir = 'incr_backup.dir'
+    uri = 'test_backup29'
+    uri2 = 'test_other'
+    value_base = '-abcdefghijkl'
+
+    few = 100
+    nentries = 5000
+
+    def parse_blkmods(self, uri):
+        meta_cursor = self.session.open_cursor('metadata:')
+        config = meta_cursor[uri]
+        meta_cursor.close()
+        # The search string will look like: 'blocks=hex)'
+        # We want just the value after the =.
+        b = re.search(',blocks=\w+', config)
```

Comment:
Try this:
```
    b = re.search(',blocks=(\w+)', config)
```
Then the 0th group is still the entire matched string, but the 1st group is the first matched paren stuff:
```
>>> b[0]
',blocks=1234'
>>> b[1]
'1234'
```
So you can just set `blocks_bitmap = b[1]` (or use `b.group(1)`)